lib/core/feature-flags/src/lib/directives/features.directive.spec.ts

```diff
@@ -20,7 +20,7 @@ import { Component } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { provideMockFeatureFlags } from '../mocks/features-service-mock.factory';
 import { FeaturesDirective } from './features.directive';
-import { By } from '@angular/platform-browser';
+import { UnitTestingUtils } from '@alfresco/adf-core';
```

Comment:
You are already in core lib, so import from @alfresco/adf-core is incorrect

Reply:
Right I missed it, fixed